lib/devise/multi_email/parent_model_extensions.rb

```diff
@@ -9,8 +9,12 @@ module ParentModelExtensions
 
       included do
         multi_email_association.include_module(EmailModelExtensions)
+
+        # Enable "autosave" to ensure email record changes are persisted
+        multi_email_association.reflection.options.merge!(autosave: true)
```

Comment:
By right new ones will be saved without the need of setting autosave, `autosave` is for auto-save those persisted records, but I don't think it's necessary here and it worked before.